include/cxxopts.hpp

```diff
@@ -1091,7 +1091,7 @@ namespace cxxopts
 
       auto riter = m_results.find(iter->second);
 
-      return riter->second.count();
+      return static_cast<int>(riter->second.count());
```

Comment:
Actually this would be better with the return type being `size_t` since it can never be negative.